apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -77,6 +77,17 @@ const buildColorExample = token => {
     return `<div class="${classes.join(' ')}" style="${cssVariable}";></div>`;
 };
 
+/**
+ * Builds the example spacing swatch to show on the token list item
+ * @param {string} token - the token value i.e. 
+ * @returns {string} - the color swatch example HTML string
+ */
+const buildSpacingExample = token => {
+    const cssVariable = `${token}px`;
+
+    return `<div class="c-tokensTable-example--spacing" style="--example-spacing: ${cssVariable}";></div>`;
```

Comment:
`--example-spacing` could be better named too. (happy to have a quick hangout and chat about, as probably easier if could see the context of it?)

Reply:
I can see it's used above too – I reckon move some of this out to the other variable above so it makes more sense in context. So:

const cssVariable = `--example-spacing: ${token}px`;